### PR TITLE
DAT-2824 Fix warnings when node has no child nodes

### PR DESCRIPTION
**Pull request by sebastianmarzjan**

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/Helpers/SimpleXmlUtil.php [hunk 1]
```diff
@@ -28,8 +28,12 @@ public static function getInstance() {
 	public function getInnerXML( $node ) {
 		$innerXML = '';
 		if ( $node instanceof \SimpleXMLElement ) {
-			foreach ( dom_import_simplexml( $node )->childNodes as $child ) {
-				$innerXML .= $child->ownerDocument->saveXML( $child );
+			$domDocument = dom_import_simplexml( $node );
+
+			if ( $domDocument->childNodes->length ) {
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Helpers/SimpleXmlUtil.php]: Maybe we could use http://php.net/manual/pl/domnode.haschildnodes.php here?

Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Helpers/SimpleXmlUtil.php]: I think we can and we will :)
Thanks!

Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Helpers/SimpleXmlUtil.php]: On second though, we'd need to have a two-sided conditional here
( if $domDocument instanceof DomNode && $domDocument->hasChildNodes() )

What do you think?

**Comment by idradm** [comment]
I'm not fond of using instanceof. Other than that looks good.

**Pull request merged**